tests/test_lambdas/test_client.py

```diff
@@ -10,13 +10,36 @@
 
 from lbz.aws_boto3 import Boto3Client
 from lbz.lambdas import LambdaClient, LambdaError, LambdaResult
+from lbz.lambdas.client import SetsEncoder
+from lbz.response import Response
 
 
 @pytest.fixture(name="lambda_client")
 def lambda_client_fixture(mocker: MockerFixture) -> MagicMock:
     return mocker.patch.object(Boto3Client, "lambda_")
 
 
+def rest_payload_factory(
```

Comment:
This factory is used only once - I don't think we should implement it this way then.